connect/service.go

```diff
@@ -0,0 +1,11 @@
+package connect
```

Comment:
I wonder do we need a new file just for this struct.

Reply:
It keeps the structure pretty predictable. We can put some utility functions for service here when needed.

Reply:
ok